lib/OpenQA/UserAgent.pm

```diff
@@ -38,6 +40,7 @@ sub new {
         for my $path (@cfgpaths) {
             my $file = $path . '/client.conf';
             next unless $file && -r $file;
+            log_info("Reading config from $file");
```

Comment:
I guess we still shouldn't log messages unconditionally in library code. And yes, it depends on the global log level but I'd rather tend to have a switch within the library. Or the library is simply able to return the path it is using to the tool which can then decide whether to log it or not.

Reply:
Maybe we can discuss it in the weekly later. From my perspective we do this all over the place in openQA library code. It's not "new". Except we have more than one config file handler so unconditional log calls do different things depending on the frontend.